lte/gateway/c/core/oai/tasks/nas5g/include/M5GCommonDefs.h

```diff
@@ -1,5 +1,12 @@
 #pragma once
 #include <arpa/inet.h>
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include "lte/gateway/c/core/oai/common/log.h"
+#ifdef __cplusplus
+}
+#endif
 #include "lte/gateway/c/core/oai/common/common_defs.h"
 #include "orc8r/gateway/c/common/logging/magma_logging.h"
```

Comment:
This should be removed

Reply:
This **M5GCommonDef.h** file is included in maximum files, if we are not including "lte/gateway/c/core/oai/common/log.h" in this file, we need to include it in all files which are having OAI logs.

Reply:
If I'm understanding this PR correctly, we should no longer have a dependency on `#include "orc8r/gateway/c/common/logging/magma_logging.h"`, right? 

But to your point, I would strongly recommend only including header files that are needed in each file. The implicit chained includes will get out of hand very quickly, from my experience.